src/suite/types.ts

```diff
@@ -100,6 +100,7 @@ export type ApplicationSuiteArgs = {
   resourcePrefix?: string;
   vpcNative?: boolean;
   migration?: MigrationArgs;
+  migrations?: { [key: string]: string[] };
```

Comment:
Instead of using string for the value let's use `MigrationArgs` to make it consistent. it will require a simple change in the implementation too

Reply:
I proposed it to be []string instead of { args: []string }. Less nesting. But we can go with MigrationArgs